server/container/metrics/metrics-es-reporter/src/test/java/org/apache/james/metric/es/ESReporterTest.java

```diff
@@ -136,7 +136,7 @@ private boolean done(ClientProvider clientProvider) {
 
     @SuppressWarnings("deprecation")
     private String getContainerIp() {
-        return esContainer.getContainerInfo().getNetworkSettings().getIpAddress();
+        return esContainer.getIp();
```

Comment:
Maybe this method call can even be removed, and *esContainer.getIp()* called directly ?